Makefile.inc

```diff
@@ -1,6 +1,8 @@
+# vim: set syntax=make:
+
 .PHONY: git-vars
 git-vars:
-ifneq ($(wildcard .git),)
+ifeq ($(shell [[ `command -v git` && -d .git ]] && echo true),true)
```

Comment:
should the output for `[[ .. ]]` be redirected to `/dev/null`?

Reply:
I guess it will output nothing in any case. The `command -v git` returns a non-empty string only if the executable is found, which will make the `[[ ... ]]` test for `-d .git` afterwards. So the output within `[[ ... ]]` will never reach `std[out,err]`.